lib/user_space/uuuid.c

```diff
@@ -36,9 +36,8 @@
 #include <fcntl.h>
 #include <unistd.h>
 
-/** path to read kmod uuid parameter */
-static const char *kmod_uuid_file = "/sys/module/m0tr/parameters/node_uuid";
-
+/** path to read node uuid parameter */
+static const char *uuid_file = "/etc/machine-id";
```

Comment:
Let's add:
```
enum {
        MACHINE_ID_LEN = 32
};

M0_BASSERT(MACHINE_ID_LEN == M0_UUID_STRLEN - 4);
```

So, we can add copy the machind id to uuid and and four "-" in the string.

Reply:
incorporated